### ISSUE TITLE
Github event notifications in channel

### ISSUE THREAD

**Issue by bbickerton** [hackerdrinks-sg/hackerdrunk_bot#4]
Hooking events from Github into the channel could help get and keep people involved with development. When an event is detected, display some sort of message in channel with relevant info/links. Should be configurable for what to alert on, which repos, etc.. Possible worthwhile events:
* hackerdrinks-sg org repos: created, deleted, archived
* Issues: created, closed, assigned, labeled, etc.
* pull requests: created, closed, merged, reviewers assigned, assigned, labeled, 
* Wiki pages: created, deleted, edited

**Comment by oswellchan** [comment]
Possible github action? https://github.com/marketplace/actions/telegram-notify

**Comment by ruiwen** [comment]
I think @davidchua has something just for this, a custom notifier

**Comment by davidchua** [comment]
Yeah this is definitely something we need to setup. I'm thinking of setting up another channel so that those interested can join in there to get notified of any changes to any of our github repositories.

That github actions looks good, we could use that.
